hw/ip/prim/doc/prim_clock_gp_mux2.md

```diff
@@ -0,0 +1,28 @@
+---
+title: "Primitive Component: Two input clock Mux with glitch protection"
+---
+
+# Overview
+`prim_clock_gp_mux2` is a two input clock mux that protects a glitch. When a current clock source is switched to the next clock source where two clocks are totally unrelated, a glitch can be generated as follows.
+
+### Glitch
+{{< wavejson >}}
+{signal: [
+  {name: 'clk0_i',           wave: '0.1....0....1....0....1....'},
+  {name: 'clk1_i',           wave: '0.1.0.1.0.1.0.1.0.1.0.1.0.1'},
+  {name: 'sel_i',            wave: '0............1.............'},
+  {name: 'clk_o',            wave: '0.1....0....101.0.1.0.1.0.1'},
+ ]}
+{{< /wavejson >}}
+
+This glitch free clock mux can avoid glitch by placing two parallel synchronizers connected to each other. 1st flop and 2nd flop are triggered by pos edge and negedge respectively to protect metastability on the sel_i signal. The following waveform shows the result. 
```

Comment:
tiny nit, `pos edge` and `negedge` :) maybe extra space on the first?

Reply:
![Screen Shot 2022-06-21 at 5 46 13 PM](https://user-images.githubusercontent.com/103145460/174919770-54840cdd-a0bb-4cf7-9d82-9ea24010ff2e.png)
Updated.

Reply:
sorry i meant the literally word `pos edge` and `nedgedge` in your description

Reply:
NP. I fixed the description. Thanks